helm/cluster-vsphere/templates/ipam/rbac.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if eq (include "isIpamEnabled" $) "true" -}}
+{{- if or (eq (include "isIpamEnabled" $) "true") (eq (include "isIpamSvcLoadBalancerEnabled" $) "true") -}}
```

Comment:
do you need that extra condition? Can `isIpamSvcLoadBalancerEnabled` exist without `isIpamEnabled` ?

Reply:
yes, they are both independent. One can have ipam for svc lb and use a "hard-coded" IP for wc's CP ip and vice-versa, or have both, or none

if at least one of those conditions is true, we run that job and pause the cluster. The job will then get the initContainers populated based on what it needs to be done. `initContainers` are used here, because the order of execution is guaranteed and then in the main container (which is run after all init containers) we just unpause the cluster. It's hard to see the whole picture from the diff, it's probably better to open a whole file w/ the job. (that bash-in-yaml spaghetti also doesn't help much ;D)

Reply:
oh and they are on or off basically if their corresponding `ipPoolName` option in helm chart values is empty or not

Reply:
In which case...





![image](https://github.com/giantswarm/cluster-vsphere/assets/19745768/2cfd275d-78d1-4779-b486-d9337d7f7853)